testing-snap/scripts/camera.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
```

Comment:
nitpick: I would add `set -eu` on the second line. Even if the script is small, if it grows, it will avoid some unecessary headaches :)

Reply:
Sure, thanks!